src/gp_cars/settings/docker.py

```diff
@@ -0,0 +1,156 @@
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.8/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+
+BASE_DIR = os.path.dirname(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '4*cr2x1kjp+tn2(drq_n@p&g9xi)d%l^qva_@dbl%-t6wxzy%h'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+TEMPLATE_DEBUG = DEBUG
+
+ALLOWED_HOSTS = []
+
+DEFAULT_EMAIL_ADDRESS = 'info@globaltrademotors.com'
+
+# admins
+ADMINS = (
+    ('Admin', DEFAULT_EMAIL_ADDRESS),
+    ('Haroon', 'haroon@sitture.com'),
+)
+
+EMAIL_HOST = 'smtp.webfaction.com'
+EMAIL_HOST_USER = 'trademotors'
+EMAIL_HOST_PASSWORD = 'sitture2'
+SERVER_EMAIL = 'info@globaltrademotors.com'
+
+# Application definition
+
+INSTALLED_APPS = (
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'django.contrib.humanize',
+    'dynamic_preferences',
+    'crispy_forms',
+    'ckeditor',
+    'imagekit',
+    'vehicles',
+    'settings',
+)
+
+MIDDLEWARE_CLASSES = (
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'django.middleware.security.SecurityMiddleware',
+)
+
+ROOT_URLCONF = 'gp_cars.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [os.path.join(
+            os.path.dirname(BASE_DIR), 'static', 'templates')
+        ],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+                'dynamic_preferences.processors.global_preferences',
+            ],
+        },
+    },
+]
+
+WSGI_APPLICATION = 'gp_cars.wsgi.application'
+
+# Database
+# https://docs.djangoproject.com/en/1.8/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+        'HOST': 'db',                      
+        'PORT': '3306', 
+        'NAME': 'trademotors',
+        'USER': 'trademotors',
+        'PASSWORD': 'sitture2',
+    }
```

Comment:
Do these now need to come from environment variables?

Reply:
I can add `RUN python manage.py test` on Dockerfile. This way the image will not be built if there are test failing.
Yes, you are right. The DB Name, user and password should go on environment variables.